src/openorganisatie/scim/models/user.py

```diff
@@ -12,6 +18,11 @@ class User(AbstractSCIMUserMixin, models.Model):
         verbose_name=_("User principal name"),
         help_text=_("User principal name van de medewerker."),
     )
+    employee_number = models.CharField(
+        max_length=100,
+        blank=True,
+        help_text=_("Unieke employeeNumber uit Entra / SCIM."),
```

Comment:
I saw from the `help_text`
```suggestion
        help_text=_("Unieke employeeNumber uit Entra / SCIM."),
        unique=True,
```

Reply:
Updated the help_text, users can have the same employee number. It's only unique in Medewerker